mne/viz/_3d.py

```diff
@@ -990,6 +1078,225 @@ def time_label(t):
+        elif not isinstance(figure, (list, tuple)):
+            figure = [figure]
+        if not all(isinstance(f, mayavi.core.scene.Scene) for f in figure):
+            raise TypeError('figure must be a mayavi scene or list of scenes')
+
+    subjects_dir = get_subjects_dir(subjects_dir=subjects_dir,
+                                    raise_error=True)
+    subject = _check_subject(stc.subject, subject, True)
+    if hemi in ['both', 'split']:
+        hemis = ['lh', 'rh']
+    else:
+        hemis = [hemi]
+
+    title = subject if len(hemis) > 1 else '%s - %s' % (subject, hemis[0])
+
+    # Plot the surface meshes
+    with warnings.catch_warnings(record=True):  # traits warnings
+        brain = Brain(subject, hemi=hemi, surf=surface, title=title,
+                      curv=False, size=size, background=background,
+                      foreground=foreground, figure=figure,
+                      subjects_dir=subjects_dir, views=views)
+
+        # Set the transparancy of the meshes
+        for row in brain._figures:
+            for fig in row:
+                for child in fig.children:
+                    props = child.children[0].children[0].actor.property
+                    props.trait_set(opacity=brain_alpha, backface_culling=True)
+
+    t = np.searchsorted(stc.times, time)
+    magnitude = abs(stc)
+
+    scale_pts, colormap, transparent = _limits_to_control_points(
+        clim, stc.data, colormap, transparent)
+
+    if scale_factor is None:
+        scale_factor = 10. / np.max(magnitude.data[:, t])
+
+    def _plot_hemi_glyphs(figure, geo):
+        """Plot the glyphs of a single hemisphere."""
+        if geo.hemi == 'lh':
+            rr = brain.geo['lh'].coords[stc.lh_vertno]
+            data = stc.lh_data
+            scalars = magnitude.lh_data[:, t]
+        else:
+            rr = brain.geo['rh'].coords[stc.rh_vertno]
+            data = stc.rh_data
+            scalars = magnitude.rh_data[:, t]
+
+        q = mayavi.mlab.quiver3d(
+            rr[:, 0],
+            rr[:, 1],
+            rr[:, 2],
+            data[:, 0, t],
+            data[:, 1, t],
+            data[:, 2, t],
+            scale_factor=scale_factor,
+            scalars=scalars,
+            colormap=colormap,
+            vmin=scale_pts[0],
+            vmax=scale_pts[2],
+            figure=figure,
+        )
+
+        # Scale colormap used for the glyphs
+        lut_table = _scale_mayavi_lut(q.parent.vector_lut_manager,
+                                      scale_pts[0], scale_pts[1],
+                                      scale_pts[2], transparent,
+                                      verbose=verbose)
+        lut_manager = q.parent.vector_lut_manager
+        lut_manager.lut.trait_set(table=lut_table)
+        lut_manager.trait_set(
+            data_range=np.array([scale_pts[0], scale_pts[2]]))
+
+        return q
+
+    # Plot the vector glyphs
+    figures = np.ravel(brain._figures)
```

Comment:
is there no equivalent public property? If so I guess it's okay to have it this way